src/fragments/lib/push-notifications/android/getting_started/40_install_lib.mdx

```diff
@@ -0,0 +1,15 @@
+Open **build.gradle (Module: app)** and add these libraries into the `dependencies` block.
+
+```groovy
+dependencies {
+    implementation 'com.amplifyframework:aws-auth-cognito:ANDROID_VERSION'
+    implementation 'com.amplifyframework:aws-push-notifications-pinpoint:ANDROID_VERSION'
+}
+```
+Click **Sync Now**.
```

Comment:
where would I click on SYnc now?

Reply:
This step is clear for Android developers and all our current documentation have this in some form.

Reply:
Yes, we mention where to add the dependencies. 

> Open **build.gradle (Module: app)** ...